VIS/Areas/VIS/Scripts/F20/wframe.js

```diff
@@ -963,6 +963,34 @@
         }
     };
 
+    AppsAction.prototype.highlightNewButton = function (highlight) {
```

Comment:
this function should be in Container class. it is re-useable for other action also.